server/controller/index.js

```diff
@@ -18,8 +18,21 @@ exports.addQuestion = async (ctx) => {
   return ctx.body;
 };
 
+exports.getAllQuestions = async (ctx) => {
+  await Question.find({})
+    .then((res) => {
+      ctx.body = res;
+      return ctx.body;
+    });
+};
+
 exports.getQuestions = async (ctx) => {
+  const { page } = ctx.request.query;
```

Comment:
We should check if `page` is a number and either return an error message or revert to a default value of 1 maybe?